pxd.c

```diff
@@ -1459,7 +1459,7 @@ static int pxd_init_disk(struct pxd_device *pxd_dev, unsigned int *blk_mq_queue_
 	pxd_dev->disk = disk;
 
 #if defined __PX_BLKMQ__ && !defined __PXD_BIO_MAKEREQ__
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,14,0) || defined(__EL9_STREAM__)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,14,0) || defined(__EL9_STREAM__) || (defined(RHEL_RELEASE_CODE) && RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(9, 7) && RHEL_RELEASE_CODE != RHEL_RELEASE_VERSION(10, 0))
 	*blk_mq_queue_flag = blk_mq_freeze_queue(q);
```

Comment:
usage of blk_mq_queue_flag was totally removed; why is this being carried further?
there is no reason for quiesce and unquiesce during volume attach, wasnt this merged before?